config.go

```diff
@@ -66,6 +66,10 @@ type Config struct {
 	//
 	// Values configured here are per-second. See zapcore.NewSampler for details.
 	Sampling *zap.SamplingConfig `toml:"sampling" json:"sampling"`
+	// ErrorOutputPath is a path to write internal logger errors to.
+	// If this field is not set, the internal logger errors will be sent to the same file as in File field.
+	// Note: if we want to output the logger errors to stderr, we can just set this field to "stderr"
+	ErrorOutputPath string
```

Comment:
I think you should provide the `toml` and `json` tag.

Reply:
@nolouch Done